scripts/export_hadiths_json.py

```
@@ -0,0 +1,55 @@
+# Fetch all hadiths from all sources
+cursor.execute(
+    """
+    SELECT h.source, h.chapter, h.chapter_no, h.hadith_no, h.text_ar, r.name as narrator_name
+    FROM hadiths h
+    LEFT JOIN hadith_chains c ON h.source = c.source AND h.chapter_no = c.chapter_no AND h.hadith_no = c.hadith_no AND c.position = 1
+    LEFT JOIN rawis r ON c.scholar_indx = r.scholar_indx
+    """
+)
+rows = cursor.fetchall()
+columns = [desc[0] for desc in cursor.description]
+all_hadiths = [dict(zip(columns, row)) for row in rows]
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)